ml/backend/ggml/ggml/.rsync-filter

```diff
@@ -13,11 +14,14 @@ include src/ggml-cuda/vendors/
 include src/ggml-cuda/template-instances/
 include src/ggml-hip/
 include src/ggml-metal/
+include CMakeLists.txt
 include *.c
 include *.h
 include *.cpp
 include *.cu
 include *.cuh
 include *.m
 include *.metal
-exclude *
+include common.cmake
+include ggml-config.cmake.in
+exclude *
```

Comment:
It looks like the newline at the end of the file is required, otherwise I get:

```
jesse@Jesses-MacBook-Pro ollama % make -f Makefile.sync sync           
sed -e 's|@FETCH_HEAD@|71e90e8813f90097701e62f7fce137d96ddf41e2|' llama/build-info.cpp.in > llama/build-info.cpp
rsync -arvzc -f "merge llama/llama.cpp/.rsync-filter" llama/vendor/ llama/llama.cpp
Transfer starting: 67 files
common/
examples/llava/
include/
src/

sent 4284 bytes  received 410 bytes  661126 bytes/sec
total size is 3277040  speedup is 698.13
rsync -arvzc -f "merge ml/backend/ggml/ggml/.rsync-filter" llama/vendor/ggml/ ml/backend/ggml/ggml
rsync: syntax error in ml/backend/ggml/ggml/.rsync-filter at entry 27
make: *** [ml/backend/ggml/ggml] Error 1
```